.github/workflows/ci-build.yml

```diff
@@ -0,0 +1,64 @@
+name: CI
+
+on:
+  push:
+    branches:
+    tags:
+  pull_request:
+
+
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
can it run on something with smaller footprint, alpine ideally

Reply:
It's not docker image. It is virtual environment, it should one of three values: macos, windows and ubuntu. See [docs](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/virtual-environments-for-github-hosted-runners)

Reply:
Oh. Then fix version to `ubuntu-18.04` so the builds are the same overtime

Reply:
Ok, i replace on `ubuntu-18.04`